lms/djangoapps/grades/tests/test_signals.py

```diff
@@ -251,3 +256,37 @@ def test_disconnect_manager_bad_arg(self):
         with self.assertRaises(ValueError):
             with disconnect_submissions_signal_receiver(PROBLEM_RAW_SCORE_CHANGED):
                 pass
+
+
+@ddt.ddt
+class UserPartitionChangeHandlerTest(TestCase):
```

Comment:
I'd suggest renaming this to something like `UserMembershipChange..` or `UserGroupingChange..`.  Since the design is no longer about integrating directly with UserPartition changes, it's theoretically an abstraction violation to call out UserPartitions here.  (As the code is checking for CohortMembership and Track changes.)